util/rom-ext-manifest-generator.py

```diff
@@ -18,15 +18,37 @@
     Directory where manifest.hjson and manifest.h.tpl reside.
 
   rom-ext-manifest-generator --output-dir:
-    Directory where manifest.hjson and manifest.h.tpl reside.
+    Directory where manifest.h will be created.
 """
 
+# The schema allows for a few different things:
```

Comment:
nit: This could be part of the module docstring. wdyt?

Reply:
done!